setup.py

```diff
@@ -32,8 +32,9 @@
     "google-api-core[grpc] >= 1.22.0, < 2.0.0dev",
     "google-cloud-core >= 1.4.1, < 2.0dev",
     "grpc-google-iam-v1 >= 0.12.3, < 0.13dev",
-    "proto-plus==1.11.0",
     "libcst >= 0.2.5",
+    "proto-plus == 1.11.0",
+    "sqlparse >= 0.3.0",
```

Comment:
`sqlparse` is only used for `dbapi` and is not required for using the Cloud Spanner client. I think it makes more sense to have this as a separate extra dependency similar to the `OpenTelemetry` dependencies under "tracing".

Reply:
The problem with making this an optional dependency is that we've got code in the library that will fail to import without it. The library still works without tracing, it just doesn't emit traces.

If we want to avoid sqlparse as a required dependency I see two options: (1) make it an optional import and catch the import errors in the DBAPI package and log an error, or (2) make `spanner_dbapi` a separate installable package in this repo, with its own set of requirements.

Reply:
As discussed I think (2) is the best option